README.md

```diff
@@ -1,5 +1,167 @@
+
+We are using the Pegasus WMS in order to connect and automatize the pipeline steps.
+
+## Folder structure
+- data
+	<br> Here is where we keep some example data files. Some of them are intermediate files generated during pipeline run. There is also a minimal set of files that defines a TVB head. These files are currently used only for tests.  
+- docs
+	<br> This folder holds some visual hints of the dependencies between the pipeline steps. For example, there is an overview diagram representing the pipeline stages. These stages are defined by scripts that can be found inside the bin folder. On the other hand, there is also an example graph diagram which is displaying the more detailed steps. This kind of diagram is automatically generated at each pipeline run.  
+- pegasus
+	<br> All the configuration files necessary for the software run are kept here. 
+	<br> At the first level there are the entry points which are explained bellow in the <b><i>Entry Point</i></b> section. 
+	<br> Inside the <i>config</i> folder, there is a folder called <i>scripts</i>. This holds bash files with calls to pipeline commands that need some specific environment configuration. These bash files should not be changed, neither their names, because they are mapped inside the workflow.
+	<br> The <i>config</i> folder also contains the configuration files which are patient specific. These are the only files that will change for each run and/or patient. The possible configurations are explained in <b>How to launch</b> section.
+- provision
+	<br> Inside this folder, you can find more details about the project dependencies.
+- tvb
+	<br> Actual python implementation is kept here. 
+	<br> Package <i>dax</i> is using the Pegasus API to define a pipeline workflow and generate the jobs graph.
+	<br> Inside <i>algo</i> package we have all the computations, services and algorithms used during the pipeline steps. Here we also have the <i>reconutils.py</i> which defines an API for calls made from the <i>config/scripts</i> bash files.
+	<br> The model classes can be found inside <i>model</i> package.
+	<br> Package <i>io</i> provides read/write functionalities from/to a variety of file formats.
+	<br> We also have a set of tests for this module and they are kept inside the <i>tests</i> package.
+
+## Data structure
+For an automated pipeline run, the patient data is divided in three categories:
+- raw data
+    <br> Usually scans, that are used as input files for the pipeline (e.g. T1, DWI, CT).
+- configurations
+    <br> This is a folder with configuration files. Each configuration file is described in the <b>How to launch</b> section. 
+    <br> This folder is patient specific. All the files should be filled in by the user or generated automatically with the <b>run_sequential.py</b> described also in <b>How to launch</b> section.
+- output data
+    <br> These are the files generated by the pipeline. The output can be structured using the configuration file <b>rc_out.txt</b>.
```

Comment:
... especially useful in case of a partial re-run.

Reply:
Moved this comment to the Configurations section because it is related to rc.txt, not rc_out.txt.